opencog/cython/opencog/atom.pyx

```diff
@@ -10,7 +10,7 @@ cdef class Atom(object):
         del self.handle
 
     def value(self):
-        return void_from_cptr(self.handle)
+        return self.handle.value()
```

Comment:
the point is, if you want to change the TV on the atom, the atom needs to be unique. otherwise, you are changing the TV on two different atoms, and they will get out of sync.   This could be a really serious issue for python -- if it really is tracking two different atoms as if they were the same, (and this is not just some stupid glitch), then the python bindings are seriously broken.  

We need to get to the bottom of this. Maybe opening a new issue for this is worth it.